docs/embedded/concepts/app-concepts/containertypes.md

```diff
@@ -7,21 +7,23 @@ ms.localizationpriority: high
 
-There are two kinds of Container Types that a developer tenant can create:
+Container Type is created and managed by the SharePoint Embedded application developer, and is strongly tied to the SharePoint Embedded application. Container Type can be created with or without linkage to an Azure billing profile, for trial/development and production purposes respectively.
 
-1. [Trial Container Types](#trial-container-types)
-1. [Standard Container Types](#standard-container-types)
+> [!NOTE]
+> You must specify the purpose of the Container Type you are creating at creation time. Dependent on the purpose, you may or may not need to provide your Azure Subscription ID. A Container Type set for trial purpose cannot be converted for production; or vice versa. 
 
-## Trial Container Types
+## Trial Container Type
 
-To enable customers to explore the development of SharePoint Embedded applications and assess its features, we offer the ability to create a Trial Container Type. Each customer can have only one Trial Container Type in their tenant at a time. The tenant developing the application can utilize Trial Container Types, which aren't linked to a billing profile. In this scenario, the partner tenant remains the same as the consuming tenant. The Trial Container Type remains valid for up to 30 days but can be removed at any time within this period. There are two ways to create a Trial Container Type:
+A Container Type can be created for trial/development purposes and is not linked to any Azure billing profile. This enables developers to explore SharePoint Embedded application development and assess its features for free. In this scenario, the developer tenant is the same as the consuming tenant.  Each developer can have only one Container Type in the Trial status in their tenant at a time; and the Container Type is valid for up to 30 days but can be removed at any time within this period. To create a Container Type for trial purposes, you can:
 
-- You can use [SharePoint Embedded Visual Studio Code Extension](../../getting-started/spembedded-for-vscode.md) to create a Trial Container Type in just a few steps. The Visual Studio Code extension also registers your Container Type and creates Containers for you.
-- A Trial Container Type can be created using SharePoint PowerShell. You must be a SharePoint Embedded Administrator or Global Administrator to run this cmdlet. If you're a SharePoint Administrator, grant yourself the SharePoint Embedded Admin role as well to execute these cmdlets.
+1. Use SharePoint Embedded Visual Studio Code Extension to create the Container Type in just a few steps. The Visual Studio Code extension also registers your Container Type and creates Containers for you.
+2. Use SharePoint PowerShell. You must be a SharePoint Embedded Administrator or Global Administrator to run this cmdlet. If you're a SharePoint Administrator, grant yourself the SharePoint Embedded Admin role as well to execute these cmdlets.
```

Comment:
Markdown numbered bullets should always use 1.... don't use explicit numbers. Let the rendering engine do the numbering for you.